HelloWorld/app/src/main/AndroidManifest.xml

```diff
@@ -0,0 +1,16 @@
+<manifest package="net.boondockradio.helloworld"
+    xmlns:android="http://schemas.android.com/apk/res/android">
+
+    <application android:allowBackup="true" android:icon="@mipmap/ic_launcher"
+        android:label="@string/app_name" android:supportsRtl="true"
+        android:theme="@style/AppTheme">
+
+        <activity android:name=".MainActivity">
+            <intent-filter>
+                <action android:name="android.intent.action.MAIN"/>
+                <category android:name="android.intent.category.LAUNCHER"/>
```

Comment:
This is a statement saying "This is the activity that the user sees when the app launches".